app/config/variables.php

```diff
@@ -402,6 +402,78 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_STORAGE_DEVICE',
+                'description' => 'Select default storage device. The default value is \'Local\'. List of supported adapters are \'Local\', \'S3\' and \'DoSpaces\'.',
+                'introduction' => '0.12.0',
```

Comment:
Let's change all `0.12.0` to `0.13.0`

Reply:
The default value is `local` not `Local`.

Reply:
Or should we switch to `Local` in all other places, either ways - lets be consistent.

Reply:
Where is it `local` ? I have set `Local` everywhere
![image](https://user-images.githubusercontent.com/6360216/145345276-08672d4f-33eb-4e1e-bb17-d4532a09cb43.png)

Reply:
It should have been `Local` everywhere, as we have constant in storage library
![image](https://user-images.githubusercontent.com/6360216/145345474-b400d532-a782-4185-bdf2-1916da507f40.png)